lms/djangoapps/courseware/models.py

```diff
@@ -145,10 +153,11 @@ def __unicode__(self):
         return unicode(repr(self))
 
 
-class StudentModuleHistory(models.Model):
+class StudentModuleHistory(CallStackMixin, models.Model):
```

Comment:
Yeah, in this case, you're trying to override a function in `models.Model`, so you have to put the `CallStackMixin` first